src/main/java/org/jenkinsci/plugins/scriptler/ScriptlerManagement.java

```diff
@@ -303,10 +303,11 @@ private GitScriptlerRepository getGitRepo() {
      * @return redirects to the repo entry page at <code>http://jenkins.orga.com/scriptler.git</code>
      * @throws IOException
      */
+    @RequirePOST
     public HttpResponse doHardResetGit() throws IOException {
         checkPermission(Hudson.ADMINISTER);
         getGitRepo().hardReset();
-        return new HttpRedirect("/scriptler.git");
+        return new HttpRedirect("../scriptler.git");
```

Comment:
Unclear why this is needed, was the direct broken before?

Reply:
If you have a context like "jenkins", so the redirect will put you from `http://localhost:8080/jenkins/scriptler.git/` to `http://localhost:8080/scriptler.git`

Reply:
Ah, I missed that there was a leading `/`. Now it's clear.